### PR TITLE
refactor(other): adjust some texts and translations

### LINKED ISSUE #2502: 🔧 [Refactor] Adjust some texts and translations
## 🔧 Refactor ticket

Adjust some texts and translations given to me by @BerndHueckstaedt .

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Adjust some texts and translations given to me by @BerndHueckstaedt .

### Issues

- fixes #2502

### Todo

- [X] None

### CODE REVIEW

File: backend/src/graphql/resolver/util/creations.test.ts [hunk 1]
```diff
@@ -183,8 +183,10 @@ describe('util/creation', () => {
         })
 
         it('has the clock set correctly', () => {
+          const targetMonth = targetDate.getMonth() + 1
+          const targetMonthString = (targetMonth < 10 ? '0' : '') + String(targetMonth)
           expect(new Date().toISOString()).toContain(
-            `${targetDate.getFullYear()}-${targetDate.getMonth() + 1}-${targetDate.getDate()}T23:`,
+            `${targetDate.getFullYear()}-${targetMonthString}-${targetDate.getDate()}T23:`,
```
Comment (Tirokk) [inline backend/src/graphql/resolver/util/creations.test.ts:189]: I had to change the test after the working ones below to get it running.

Comment (ulfgebhardt) [reply backend/src/graphql/resolver/util/creations.test.ts:189]: I fixed this in https://github.com/gradido/gradido/pull/2505
lets separate this?

File: frontend/src/components/ContentFooter.spec.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ describe('ContentFooter', () => {
       })
 
       it('renders the copyright year', () => {
-        expect(mocks.$t).toBeCalledWith('footer.copyright.year', { year: 2022 })
+        expect(mocks.$t).toBeCalledWith('footer.copyright.year', { year: 2023 })
```
Comment (ulfgebhardt) [inline frontend/src/components/ContentFooter.spec.js:36]: I fixed this in https://github.com/gradido/gradido/pull/2505

**Review by ulfgebhardt — COMMENTED** [review]
This is good for me, but I'd like to separate the test fixing from the text changes - lets discuss this.

**Review by ulfgebhardt — APPROVED** [review]

File: frontend/src/components/ContentFooter.spec.js (hunk 2 above)
Comment (Tirokk) [reply frontend/src/components/ContentFooter.spec.js]: Solved there

**Pull request merged**